_posts/en/2016-xx-xx-easy-foreach-operation.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+
+title: Easy forEach operation
+tip-number: xx
+tip-username: sekaiamber
+tip-username-profile: https://github.com/sekaiamber
+tip-tldr: Make an easy `forEach` operation by using a simple regular expression.
+
+categories:
+    - en
+---
+
+We can use a simple regular expression `/[^, ]+/g` to make an easy `forEach` operation.  
+For example, in jQuery, we can find a lot of DOM manipulations in pairs, like `append` and `appendTo`, they are just exchange the matched elements and target elements. So, we can code like this:
```

Comment:
I'm not a fan of using jQuery for tips. If there is a specific jQuery issue, solved via a tip, I'm fine with that (e.g there is an angularJS related tip).